### PR TITLE
feat: rules page

### PR DESCRIPTION
**Pull request by vrx29**
Rules page for quiz app.

Live Preview : https://gimesto-quiz-git-rules-vrx29.vercel.app/

### CODE REVIEW

File: src/pages/Rules/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            <span>
+              <FingerRight className="icon-arrow" />
+            </span>
+            There will be 5 questions.
+          </li>
+          <li>
+            <span>
+              <FingerRight className="icon-arrow" />
+            </span>
+            Each question will be of one mark.
```
Comment (hsnice16) [inline src/pages/Rules/index.tsx]: Here, I think, you can map over an array as only the rules text is changing.
Something like this,
```javascript
["There will be  5 questions", "Each question will be of one mark", ...other rules].map((rule) => (

<li>
     <span>
              <FingerRight className="icon-arrow" />
            </span>
            {rule}
</li>

))
```

In this way, number of lines in the file will reduce.

**Review by hsnice16 — APPROVED** [review]
Great work. UI is superb. Just added a suggestion that I noticed.

File: src/pages/Rules/index.tsx (hunk 1 above)
Comment (vrx29) [reply src/pages/Rules/index.tsx]: ok will make an array for storing rules

**Review by anik31 — APPROVED** [review]
Nice work @vrx29 , the UI looks good. Code is very clean & easily readable. Nothing to suggest.

**Pull request merged**